src/probnum/prob/distributions/distribution.py

```diff
@@ -229,7 +263,7 @@ def sample(self, size=()):
             return self._sample(size=size)
         else:
             raise NotImplementedError(
-                'The function \'sample\' is not implemented for object of class {}'.format(type(self).__name__))
+                'The function \'sample\' is not implemented for object of class {}.'.format(type(self).__name__))
```

Comment:
maybe extract these kinds of error into an `errormsg` variable? That makes the line automatically shorter and more readable.

Reply:
Following `scipy` convention here which is: Custom error type for often repeated errors of a specific form, otherwise (potentially repeated) standard error messages.